htmd/__init__.py

```diff
@@ -20,14 +20,15 @@
 from htmd.projections.kmeanstri import KMeansTri
 from htmd.userinterface import UserInterface
 from htmd.molecule.molecule import Molecule
-from htmd.adaptive.adaptiverun import AdaptiveRun
+from htmd.adaptive.adaptiverun import AdaptiveRun, AdaptiveRunNew
```

Comment:
We are removing the old one and renaming New to AdaptiveRun. No need for this line.